### ISSUE TITLE
internalServerError should receive the exception

### ISSUE THREAD

**Issue by cfsilence** [perwendel/spark#829]
Could you pass a third argument to `internalServerError()` containing the original exception object the same way `exception()` receives it?

**Comment by tipsy** [comment]
Would be happy to merge a PR!

**Comment by cfsilence** [comment]
I'll submit something when I get a free moment.  Thanks!

**Comment by cfsilence** [comment]
I've not done a pull request before - do you have any advice on how to do one?  Should I branch master, make the fix and then commit the branch?

**Comment by tipsy** [comment]
Yup, sounds about right. GitHub has a nice tutorial on it: https://help.github.com/articles/creating-a-pull-request/

**Comment by cfsilence** [comment]
OK, I've pulled master, created a branch, patched the issue and ran all the tests, but I can not push my branch (403 error).  Do I need permissions granted to push the branch?

**Comment by cfsilence** [comment]
Disregard.  I figured it out.  Let me know if you need any changes, but all tests are passing and I've confirmed it works as expected in a sample application locally.
